worlds/dark_souls_3/__init__.py

```diff
@@ -0,0 +1,280 @@
+        items_address = []
+        locations_id = []
+        locations_address = []
+        locations_target = []
+        for location in self.world.get_filled_locations():
+            if location.item.player == self.player:
+                items_id.append(location.item.code)
+                items_address.append(item_dictionary[location.item.name])
+
+            if location.player == self.player:
+                locations_address.append(location_dictionary_table[location.name])
+                locations_id.append(location.address)
+                if location.item.player == self.player:
+                    locations_target.append(item_dictionary[location.item.name])
+                else:
+                    locations_target.append(0)
+
+        data = {
+            "options": {
+                "auto_equip": self.world.auto_equip[self.player].value,
+                "lock_equip": self.world.lock_equip[self.player].value,
+                "no_weapon_requirements": self.world.no_weapon_requirements[self.player].value,
+            },
+            "seed": self.world.seed_name,  # to verify the server's multiworld
+            "slot": self.world.player_name[self.player],  # to connect to server
+            "base_id": self.base_id,  # to merge location and items lists
+            "locationsId": locations_id,
+            "locationsAddress": locations_address,
+            "locationsTarget": locations_target,
+            "itemsId": items_id,
+            "itemsAddress": items_address
+        }
+
+        # generate the file
+        filename = f"AP-{self.world.seed_name}-P{self.player}-{self.world.player_name[self.player]}.json"
+        with open(os.path.join(output_directory, filename), 'w') as outfile:
+            json.dump(data, outfile)
+
+    game: str = "Dark Souls III"  # name of the game/world
+    options = dark_souls_options  # options the player can set
+    topology_present: bool = True  # show path to required location checks in spoiler
+    remote_items: bool = False  # True if all items come from the server
+    remote_start_inventory: bool = False  # True if start inventory comes from the server
+
+    # data_version is used to signal that items, locations or their names
+    # changed. Set this to 0 during development so other games' clients do not
+    # cache any texts, then increase by 1 for each release that makes changes.
+    data_version = 1
+
+    # ID of first item and location, could be hard-coded but code may be easier
+    # to read with this as a property.
+    base_id = 100000
+
+    # The following two dicts are required for the generation to know which
+    # items exist. They could be generated from json or something else. They can
+    # include events, but don't have to since events will be placed manually.
+    item_name_to_id = {name: id for id, name in enumerate(item_dictionary_table, base_id)}
+    location_name_to_id = {name: id for id, name in enumerate(location_dictionary_table, base_id)}
```

Comment:
Please put the static class members at the beginning of your class

Reply:
Fixed